### ISSUE TITLE
fix lab music; not two expections but one in first task

### ISSUE THREAD

**Issue by bjornregnell** [lunduniversity/introprog#330]
The text about what exceptions can occur is false (e.g. NuberFormatException can never occur in fromString in Pitch). Check and fix text, probably only one Exception is possible???

**Comment by jos-rosenqvist-3812** [comment]
It's possible to get a `NumberFormatException` if you write a number that's too large to fit in an `Int`, but that's a rather obscure case that I wouldn't expect the majority of students to be able to come up with.

**Comment by jos-rosenqvist-3812** [comment]
Maybe we could solve this by having the code split at the first digit instead of using `partition`?

**Comment by ErikPrantare** [comment]
I agree with splitting at the first digit. I think using `span` instead of `partition` seems more natural.

**Comment by jos-rosenqvist-3812** [comment]
This afternoon, I noticed that there actually are two possible exceptions in `fromString` even if `NumberFormatException` isn't counted. (I won't say which they are here on GitHub since that would give away the solution for the task.)

**Comment by bjornregnell** [comment]
music is now a project; help on PR to fix above is welcome...
